public/content/de/welcome/install-d-locally.md

```diff
@@ -0,0 +1,45 @@
+# D lokal installieren
+
+Auf der [Website der Programmiersprache](https://dlang.org) wird stets die neuste
+Version des Referenzkompilers **DMD** (Digital Mars D) zum [Download](http://dlang.org/download.html)
+angeboten und kann wie folgt installiert werden:
+
+### Windows
+
+* [Installer](http://downloads.dlang.org/releases/2.x/2.071.0/dmd-2.071.0.exe)
+* oder: [Archive](http://downloads.dlang.org/releases/2.x/2.071.0/dmd.2.071.0.windows.7z)
+* mit [chocolatey](https://chocolatey.org/packages/dmd): `choco install dmd`
+
+### Mac OS X
+
+* `.dmg` [package](http://downloads.dlang.org/releases/2.x/2.071.0/dmd.2.071.0.dmg)
+* oder: [Archive](http://downloads.dlang.org/releases/2.x/2.071.0/dmd.2.071.0.osx.tar.xz)
+* mit [Homebrew](http://brew.sh): `brew install dmd`
+
+### Linux / FreeBSD
+
+DMD kann am schnellsten durch Ausführen von `curl -fsS https://dlang.org/install.sh | bash -s dmd`
+in dem Benutzerverzeichnis installiert werden.
```

Comment:
> im Benutzerverzeichnis (aber bloße Stilfrage)

Fair enough, I don't mind at all and considering you took the time to notice & write, I will change it ;-)